db/migrate/20160205141721_remove_newest_redirects_table.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveNewestRedirectsTable < ActiveRecord::Migration
+  def up
+    drop_table :newest_redirects
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration, "newest_redirect table can't be recreated"
```

Comment:
This looks good to me and consistent with the general rails guidance. :+1: 

Although your second argument should pluralise the table name.